dayFive/changeMin.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = function(currentMin : number, currentMax : number) : number {
```

Comment:
Check out es6 exports
```suggestion
export const changeMin = function(currentMin : number, currentMax : number) : number {
```